src/main/scala/uk/gov/nationalarchives/tdr/api/http/Routes.scala

```diff
@@ -23,6 +25,29 @@ class Routes(val config: Config) extends Cors {
   val ttlSeconds: Int = 10
```

Comment:
I just noticed the Logger name is wrong on line 24. ☝️ 

`Logger(classOf[Routes])` is [probably the thing to use](https://github.com/lightbend/scala-logging#using-scala-logging), because it includes the package name as well as the class name.